js/crypto.js

```diff
@@ -156,11 +162,117 @@ function generateJSON(el) {
     el.setAttribute('download', 'Crypto-Touchbar-App-' + selectedFiatObj.ticker + '.json');
 }
 
+function addCustomCoin(customCoin) {
+    const cryptoSelector = document.createElement('div'),
+    element = document.createElement('input'),
+    text = document.createElement('label'),
+    colour = document.createElement('button'),
+    icon = document.createElement('img'),
+    iconCanv = document.createElement('canvas');
+
+    element.type = 'checkbox';
+    element.value = customCoin.value;
+    element.addEventListener('change', addCrypto, false);
```

Comment:
Is the `false` needed?

Reply:
think this is a hangover from misunderstanding how to pass in parameters to the event. now handled by `element.someParam = value` . Consider it gone.